src/components/post-card.astro

```diff
@@ -0,0 +1,85 @@
+---
+import { Image } from 'astro:assets';
+import type { CollectionEntry } from 'astro:content';
+import formatHumanDate from 'src/utils';
+
+export type Props = CollectionEntry<'post'>;
+
+const { data, slug } = Astro.props;
+const humanDate = formatHumanDate(data.date);
+---
+
+<a href={data.sameAs ? data.sameAs[0] : `/artikler/${slug}`}>
+  <article>
+    <Image src={data.image} alt={data.title} class="project-image" />
+    <header>
+      <time datetime={data.date.toISOString()}>
+        {humanDate}
+      </time>
+      <div class="description">
+        <h2>{data.title}</h2>
+        <p>{data.description}</p>
+      </div>
+    </header>
+  </article>
+</a>
+
+<style>
+  a {
+    display: block;
+    color: inherit;
+    text-decoration: inherit;
+    outline: 0;
+  }
+
+  a:hover {
+    color: inherit;
+  }
+
+  .project-image {
+    width: 100%;
+    aspect-ratio: 11/7;
+    height: auto;
+    object-fit: cover;
+  }
+
+  .project-tags {
+    display: flex;
+    gap: 0.5rem;
+    margin-block-end: 0.5rem;
+  }
+
+  .project-tags span {
+    padding: 0.25rem 0.5rem;
+    border-radius: 2rem;
+    background-color: hsl(var(--blue-base) 10%);
+    color: white;
+    font-size: 0.75rem;
+    font-weight: 500;
+  }
```

Comment:
Styling for .project-tags and .project-tags span is defined, but never used. Consider removing this if not needed in the future.

```suggestion
```